tft.py

```diff
@@ -864,7 +830,7 @@ def main():
 
     if not league_api_integration.get_lcu_process():
         logger.warning("League client is not open, attempting to start it")
-        league_directory = system_helpers.determine_league_install_location(CONFIG["OVERRIDE_INSTALL_DIR"])
+        league_directory = system_helpers.determine_league_install_location(config.get_override_install_location())

```

Comment:
This may benefit from being explicit about what's being passed, just since the function name seems a bit funky as a parameter? 🤔 
```suggestion
        league_directory = system_helpers.determine_league_install_location(override_path=config.get_override_install_location())
```

Again, not strongly feeling about this one, it's just a bit of a code smell but I'm not sure what the answer is because the function name itself makes sense, just in this context it feels weird.

Reply:
I like making arguments explicit. But since the methods in `config` are globally available, we could even make it so that the override isn't passed as an argument but just part of the determine location method, which is even cleaner, IMO. What do you think about that alternative?

Reply:
Tbh that works fine for me, I can't see a reason why we'd need to set this "manually" from a method argument when we can pull it from the config module directly! 👌